packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -225,6 +226,35 @@ function dailyDungeon(additionalReady: () => boolean) {
   };
 }
 
+function lavaDogs(additionalReady: () => boolean) {
+  return {
+    completed: () => lavaDogsComplete(),
+    ready: () => additionalReady() && shouldLavaDogs(),
+    prepare: () => {
+      if (
+        garboValue($item`superheated metal`) * 0.95 +
+          (!get("_volcanoSuperduperheatedMetal")
+            ? garboValue($item`superduperheated metal`) * 0.05
+            : garboValue($item`superheated metal`) * 0.05) >
+        mallPrice($item`heat-resistant sheet metal`)
+      ) {
+        acquire(
+          1,
+          $item`heat-resistant sheet metal`,
+          garboValue($item`superheated metal`),
+        );
+      }
+    },
```

Comment:
```suggestion
    prepare: () => {
      const metalValue = get("_volcanoSuperduperheatedMetal") ? garboValue($item`superheated metal`) : sum([[$item`superheated metal`, 0.95], [$item`superduperheated metal`, 0.05]], ([item, rate]) => rate * garboValue(item));
      if (metalValue > mallPrice($item`heat-resistant sheet metal`)) {
        acquire(
          1,
          $item`heat-resistant sheet metal`,
          metalValue,
        );
      }
    },
```

Reply:
This'll fail linting at first because it needs to be broken up, so you can pull the changes and lint and then push :-/